MultipleImages/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import uuid from 'uuid';
+import styles from './style.css';
+import Image from '../Image';
+
+const getUrlAmount = (urls) => {
+  if (urls.length <= 2) {
```

Comment:
Great catch on this one! My initial thought was that it's grabbing the number of URLs so that it can apply a certain style and taking another look, it totally doesn't describe what it's *actually* doing! Awesome! Gonna update this function name to be `getImageWrapperHeight` -- love that name. 👍